[2022-02-09 08:58:56,187] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 08:58:56,201] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 08:58:56,202] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:58:56,202] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:58:56,202] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:58:56,217] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-05-01 23:00:00+00:00
[2022-02-09 08:58:56,223] {standard_task_runner.py:52} INFO - Started process 140 to run task
[2022-02-09 08:58:56,228] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-05-01T23:00:00+00:00', '--job-id', '454', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpyzqi9iyf', '--error-file', '/tmp/tmpmyeb6o_h']
[2022-02-09 08:58:56,229] {standard_task_runner.py:77} INFO - Job 454: Subtask bigquery_external_table_task
[2022-02-09 08:58:56,298] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-05-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:58:56,344] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:58:56,368] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-01T23:00:00+00:00
[2022-02-09 08:58:56,370] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 08:58:57,399] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20200501T230000, start_date=20220209T085856, end_date=20220209T085857
[2022-02-09 08:58:57,446] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:58:57,484] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:14:06,061] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:06,079] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:06,079] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:06,079] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:14:06,079] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:06,103] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-05-01 23:00:00+00:00
[2022-02-09 09:14:06,113] {standard_task_runner.py:52} INFO - Started process 209 to run task
[2022-02-09 09:14:06,123] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-05-01T23:00:00+00:00', '--job-id', '543', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpe2drv_ei', '--error-file', '/tmp/tmp_n4mkl65']
[2022-02-09 09:14:06,125] {standard_task_runner.py:77} INFO - Job 543: Subtask bigquery_external_table_task
[2022-02-09 09:14:06,247] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-05-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:14:06,331] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:14:06,376] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-01T23:00:00+00:00
[2022-02-09 09:14:06,378] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:14:07,475] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20200501T230000, start_date=20220209T091406, end_date=20220209T091407
[2022-02-09 09:14:07,502] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:14:07,539] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
